.github/CONTRIBUTING.md

```diff
@@ -1,173 +1,221 @@
+- check if the channel is working outside your country. You can use services like [streamtest.in](https://streamtest.in/) to do this.
+- find out from which country the channel is being broadcasted. This information can usually be found on [lyngsat.com](https://www.lyngsat.com/search.html) or [wikipedia.org](https://www.wikipedia.org/)
+- find the corresponding ISO_3166-2 code for the country. You can find a full list of codes here: https://en.wikipedia.org/wiki/ISO_3166-2
+- open the `/channels` folder and find the file that has the same code in its name and open it
+- find the broken link in this file
+- replace it with working one
+- commit all changes
+- send a pull request
 
-If everything is fine, then you can proceed to add the channel to the playlist. 
+**Do you want to report a channel that is not on the playlist?**
 
-If you know exactly in which country this channel is broadcast, then simply select the appropriate playlist from the `channels/` folder and add the channel to it.
+Create an [issue](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+request&template=channel-request.md&title=Add%3A+xxx) with a description of the channel (**IMPORTANT:** an issue should contain a request for only one channel, otherwise it will be closed immediately).
 
-If you are not sure which playlist to add this channel to, you can add it to the `channels/unsorted.m3u` file. It was created specifically for such cases.
+**Would you like to add a new channel to playlist?**
 
-Further on the format of links in the playlist. For the broadcast to work in the playlist, you need to add at least minimal information about the channel, like so:
+- make sure that the link you want to add works. It is recommended to use [VLC media player](https://www.videolan.org/vlc/index.html) for this
+- check if the channel is working outside your country. You can use services like [streamtest.in](https://streamtest.in/) to do this
+- find out the full name of the channel and from which country it is broadcasting. This information can usually be found on [lyngsat.com](https://www.lyngsat.com/search.html) or [wikipedia.org](https://www.wikipedia.org/)
+- find the corresponding ISO_3166-2 code for the country. You can find a full list of codes here: https://en.wikipedia.org/wiki/ISO_3166-2
+- open the `channels/` folder and find a file with the same name as the country code
+- at the very end of this file add a link to the channel with a description
+- commit all changes
+- send a pull request
 
-```xml
-#EXTINF:-1,Example TV
-http://example.com/stream.m3u8
-```
-
-But of course, the more channel information you add, the better. Here's an example of what a full version of a link in a playlist might look like:
+In case you were unable to determine which country the channel belongs to, you can put a link to it in the `channels/unsorted.m3u`.
```

Comment:
```suggestion
In case you were unable to determine which country the channel belongs to, add the channel onto the `channels/unsorted.m3u` playlist.
```